src/Picqer/Financials/Exact/Connection.php

```diff
@@ -464,25 +464,30 @@ private function acquireAccessToken()
         }
     }
 
-    private function getDateTimeFromExpires($expires)
+    /**
+     * Translates expires_in to a Unix timestamp.
+     * @param int $expiresIn Number of seconds until the token expires.
```

Comment:
Sorry, slight nuance - the value returned in the JSON is a `string` (which is also what `ctype_digit()` needs, as feeding it `int`s may yield unexpected results; see http://php.net/manual/en/function.ctype-digit.php#example-6311)